crates/wallet/src/wallet/coin_selection.rs

```diff
@@ -1090,13 +1100,12 @@ mod test {
     }
 
     #[test]
+    #[ignore]
```

Comment:
❓ Why are these tests are ignored?

Reply:
To avoid adding `RngCore` to the `CoinSelection` trait, `single_random_draw` was moved into its own function. From the code comment:
>  // In this case bnb won't find a suitable match and single random draw will select three outputs
So this test does not actually test "bnb_coin_selection", instead it tests that the case in which BnB fails defaults back to single random draw silently. In this PR the BnB no longer fails silently, and single random draw is called explicitly in the error case. See `crates/wallet/src/wallet/coin_selection.rs` line 520 in the diff. I need to take my own advice and add the ignore reason. Either that or we remove it.

For the second one, I think there is no exact match, and the BnB previously failed back to SRD, but I will double check.

Reply:
Added `ignore` reasons